wsagent/agent/src/main/resources/org.eclipse.che.ws-agent.script.sh

```diff
@@ -52,8 +52,8 @@ INSTALL_JDK=false
 command -v ${JAVA_HOME}/bin/java >/dev/null 2>&1 || {
     INSTALL_JDK=true;
 } && {
-    java_version=$(${JAVA_HOME}/bin/java -version 2>&1 | sed 's/.* version "\\(.*\\)\\.\\(.*\\)\\..*"/\\1\\2/; 1q')
-    if [ ! -z "${java_version##*[!0-9]*}" ] && [ "${java_version}" -lt "18" ]; then
+    java_version=$(${JAVA_HOME}/bin/java -version 2>&1 | grep version  | awk '{print $NF}' | sed 's/"//g' | cut -d '.' -f2)
```

Comment:
Library images for alpine, busybox, ubuntu, debian, centos, fedora and rhel have awk.

Also, in all other agents we use awk - so far so good. So, if we are to check if awk is available, then we should do it across all agent scripts. However, having a custom image without awk is quite uncommon I think (maybe some super stripped images)